src/components/ValidatorView.js

```diff
@@ -37,6 +37,18 @@ function ValidatorViewController($scope, $rootScope, $log, $http, $window, $q, $
         className: 'alert-warning',
         content_tid: 'WARNING_V5_UNOFFICIAL'
       }]
+    },
+    'draft-06': {
+      service: validatorFactoryAJV('draft-06'),
+      name: 'draft-06 (latest)'
+    },
+    'Experimental': {
```

Comment:
Please can we use lower case `experimental`, rather than `Experimental`?  Or did you have a particular rationale?